app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,81 @@
+class MultiValueInput < SimpleForm::Inputs::CollectionInput
+
+  def input(wrapper_options)
+    @rendered_first_element = false
+    input_html_classes.unshift("string")
+    #input_html_options[:type] ||= 'text'
+    input_html_options[:name] ||= "#{object_name}[#{attribute_name}][]"
+
+    markup = <<-HTML
+
+
+        <ul class="listing">
+    HTML
+
+    collection.each_with_index do |value, i|
+      unless value.to_s.strip.blank?
+        markup << <<-HTML
+          <li class="field-wrapper">
+            #{build_field(value)}
+          </li>
+        HTML
+      end
+    end
+
+    markup << <<-HTML
+          <li class="field-wrapper">
+            #{build_field('')}
+          </li>
+        </ul>
+
+    HTML
+  end
+
+
+  private
+
+  def build_field(value)
```

Comment:
This method is a bit beefy -- I might like to see it broken down a bit, perhaps into smaller methods, each of which does a single thing.